src/jit/importer.cpp

```diff
@@ -13594,7 +13594,7 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                         op1->gtFlags |= GTF_ORDER_SIDEEFF; // Prevent this from being reordered
                         op1->gtFlags |= GTF_IND_VOLATILE;
```

Comment:
It seems unnecessary to me to set these two flags when we see a UNALIGNED prefix, but that isn't part of your change.